src/client/components/App.js

```diff
@@ -1,12 +1,35 @@
 // Imports
 import React from 'react'
 import { Link, Route, BrowserRouter } from 'react-router-dom'
+import styled from 'styled-components'
 
 // App Imports
 import Header from './Header'
 import Sidebar from './Sidebar'
 import Editor from './Editor'
 
+// Styles
+const AppWrapper = styled.div`
```

Comment:
Is this just a hook for styling you'll apply later, or are you standardizing around generating major UI markup as components (whether styled or normal React)?

Reply:
Yeah I saw that too -- I was screwing around at one point trying to get the application to fill 100% of the height, and this was part of the "smash my face into the problem" solutionin (there used to be styles in this element)

I forgot to delete it; I should probably go ahead and revert back to the <div> for now.

Reply:
Ah got it. Didja solve the 100% height or need a hand?